spec/helpers.lua

```diff
@@ -3133,6 +3133,56 @@ do
   end
 end
 
+---
+-- Assertion to partially compare two lua tables.
+-- @function partial_match
+-- @param partial_table the table with subset of fields expect to match
+-- @param full_table the full table that should contain partial_table and potentially other fields
+local function partial_match(state, arguments)
```

Comment:
A potentially helpful function but **_definitely_** needs tests to ensure validity.